actor/kvcounter-wasmcon2023/Cargo.toml

```diff
@@ -17,7 +17,7 @@ mime_guess = "2.0.4"
 rust-embed = "6.8.1"
 serde = { version = "1.0.186", features = ["derive"] }
 serde_json = { version = "1.0.105" }
-wit-bindgen = { git = "https://github.com/bytecodealliance/wit-bindgen.git", rev = "a15cb9dbee4f619b286fc22879a94f7197d55b18" }
+wit-bindgen = { git = "https://github.com/bytecodealliance/wit-bindgen.git", rev = "749c01697bb3b11daeae4225789e14b765dcf839" }
```

Comment:
is there a reason not to use the latest release here?

Reply:
I just wanted to pin it to a revision I was sure worked. I haven’t tested latest at all and didn’t want to risk it this close to wasmcon

Reply:
I don't understand the issue here, especially since this appears to be just an arbitrary rev, but whatever works.
As a point of reference, https://github.com/wasmCloud/wasmCloud has only been using latest, released `wit-bindgen` versions for months with no issues whatsoever.
Talking about WasmCon, let's please make sure that users use official releases and not arbitrary git revs.

Reply:
I gave it a whirl with 0.11 and got it to work